### PR TITLE
fix(breakpoint): add custom breakpoints

### PR DESCRIPTION
**Pull request by daniarsuprianto**
BREAKING CHANGE: values for all breakpoints are updated to match funda bp

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -83,6 +83,12 @@ module.exports = {
                 '2/3': '66%',
             },
         },
+      screens: {
```
Comment (sukalas-a) [inline index.js:86]: What about:
s, m , l, xl ?

Comment (daniarsuprianto) [reply index.js:86]: im okay with that.. just try to minimise the breaking change. but if you guys agree then i can change it to s m l xl

Comment (sukalas-a) [reply index.js:86]: Isn't it either way a breaking change?
I suggesting it just because it looks a bit simpler. Both are clear

Comment (daniarsuprianto) [reply index.js:86]: indeed either way is breaking change, but at least, you don't need to search in the code for sm md lg and replace it with s m l.. the breaking change will only affect the min-width value.

Comment (sukalas-a) [reply index.js:86]: We need to replace the fd- prefix as well right?

**Review by sukalas-a — APPROVED** [review]

**Pull request merged**